src/styles/main.scss

```diff
@@ -1,15 +1,6 @@
-html {
-  font-family: Roboto, sans-serif;
-}
-
-body {
-  margin: 0;
-}
-
-* {
-  box-sizing: border-box;
-}
-
-@import "../styles/utils/variables";
+@import "utils/reset";
+@import "utils/normalize";
+@import "utils/variables";
+@import "utils/extends";
 @import "_header.scss";
 @import "_catalog.scss";
```

Comment:
Better store blocks in the `blocks` folder. Also, you can omit `_` and extension in imports.

Reply:
> Better store blocks in the `blocks` folder. Also, you can omit `_` and extension in imports.

Done